src/kartograf/tests/test_ring_filters.py

```diff
@@ -85,3 +85,26 @@ def test_whole_rings_safe():
     newmapping = filters.filter_whole_rings_only(m1, m2, mapping)
 
     assert newmapping == mapping
+
+
+@pytest.mark.parametrize('molA_B_exp_mapping', [
+    (Chem.MolFromSmiles("C1CCCC2C1CCCC2"), Chem.MolFromSmiles(
+        "C1CCCC2C1CCCC2"), {i:i for i in range(10)}),
+    (Chem.MolFromSmiles("c1cccc2c1cccc2"), Chem.MolFromSmiles(
+        "c1cccc2c1cccc2"), {i: i for i in range(10)}),
+    (Chem.MolFromSmiles("C1CCCc2c1cccc2"), Chem.MolFromSmiles(
+        "C1CCCC2C1CCCC2"), {i: i for i in range(6)}),
+    (Chem.MolFromSmiles("C1CCCC2C1cccc2"), Chem.MolFromSmiles(
+        "C1CCCC2C1CCCC2"), {i: i for i in range(10)}),
+    (Chem.MolFromSmiles("c1cccc2c1CCCC2"), Chem.MolFromSmiles(
+        "C1CCCC2C1CCCC2"), {i: i for i in range(4,10)}),
+])
```

Comment:
can you add a comment on which individual test case is testing?  E.g. writing here that this is looking at a fused ring case and the expected output is that one ring is kept that would help